s27125-HubertJozwiak/zajecia3/cwiczenia1/zad4.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ $(ls | wc -l) -gt 5 ]; then
```

Comment:
Tu jest błąd. Jeśli w katalogu będą inne obiekty tj. linki symboliczne, katalogi, kolejki itd... to wtedy ten algorytm się wysypie.